source/plugin/tab-lists.rst

```diff
@@ -0,0 +1,73 @@
+=========
+Tab Lists
+=========
+
+Tab lists are used in Minecraft to display the list of players currently on a server. The SpongeAPI allows for
+manipulation of the tab list on a per-player basis.
+
+To get a player's ``TabList``, you simply need to call the ``getTabList()`` method off of the ``Player`` who's
+``TabList`` you wish to modify:
+
+.. code-block:: java
+    
+    import org.spongepowered.api.entity.living.player.Player;
+    import org.spongepowered.api.entity.living.player.tab.TabList;
+    
+    TabList tablist = player.getTabList();
+
+Now that we have obtained the ``TabList``, we can modify several components of it. For example, to set the header and
+footer of the ``TabList``, we simply need to call their appropriate methods:
+
+.. code-block:: java
+    
+    import org.spongepowered.api.text.Text;
+    import org.spongepowered.api.text.format.TextColors;
+    
+    tablist.setHeader(Text.of(TextColors.GOLD, "The tab list header"));
+    tablist.setFooter(Text.of(TextColors.RED, "The tab list footer"));
+
+Or we can call the ``setHeaderAndFooter()`` method:
```

Comment:
It should be documented that calling `setHeaderAndFooter` is preferred when changing both, as it sends one packet instead of two.